includes/class-type-registry.php

```diff
@@ -45,6 +45,18 @@ public function init( \WPGraphQL\Registry\TypeRegistry $type_registry ) {
 			\WPGraphQL\QL_Events\Type\WPObject\WooAttendee_Type::register_fields();
 			\WPGraphQL\QL_Events\Type\WPObject\Ticket_Linked_Data_Type::register();
 
+			// Custom ticket meta
+			\WPGraphQL\QL_Events\Type\WPInterface\Ticket_Field::register_type();
+			\WPGraphQL\QL_Events\Type\WPObject\Ticket_Field\Birthdate::register();
+			\WPGraphQL\QL_Events\Type\WPObject\Ticket_Field\Checkbox::register();
+			\WPGraphQL\QL_Events\Type\WPObject\Ticket_Field\Date::register();
+			\WPGraphQL\QL_Events\Type\WPObject\Ticket_Field\Dropdown::register();
+			\WPGraphQL\QL_Events\Type\WPObject\Ticket_Field\Email::register();
+			\WPGraphQL\QL_Events\Type\WPObject\Ticket_Field\Phone::register();
+			\WPGraphQL\QL_Events\Type\WPObject\Ticket_Field\Radio::register();
+			\WPGraphQL\QL_Events\Type\WPObject\Ticket_Field\Text::register();
+			\WPGraphQL\QL_Events\Type\WPObject\Ticket_Field\URL::register();
```

Comment:
We could simplify this down to the following, due to the namespace at the top of the file:

```suggestion
			Type\WPInterface\Ticket_Field::register_type();
			Type\WPObject\Ticket_Field\Birthdate::register();
			Type\WPObject\Ticket_Field\Checkbox::register();
			Type\WPObject\Ticket_Field\Date::register();
			Type\WPObject\Ticket_Field\Dropdown::register();
			Type\WPObject\Ticket_Field\Email::register();
			Type\WPObject\Ticket_Field\Phone::register();
			Type\WPObject\Ticket_Field\Radio::register();
			Type\WPObject\Ticket_Field\Text::register();
			Type\WPObject\Ticket_Field\URL::register();
```

Reply:
Or even add `use WPGraphQL\QL_Events\Type\WPObject\Ticket_Field;` at the top then change to:

```suggestion
			Ticket_Field::register_type();
			Ticket_Field\Birthdate::register();
			Ticket_Field\Checkbox::register();
			Ticket_Field\Date::register();
			Ticket_Field\Dropdown::register();
			Ticket_Field\Email::register();
			Ticket_Field\Phone::register();
			Ticket_Field\Radio::register();
			Ticket_Field\Text::register();
			Ticket_Field\URL::register();
```